<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Skylark.local" name="hub.test.RegressionSuite.UserMyAccount" tests="21" failures="1" timestamp="7 Nov 2014 07:11:46 GMT" time="485.307" errors="0">
  <testcase name="RT_04436_MyAccount_Branch_ID_Not_Null" time="6.526" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02545_MyAccount_Duplicate_Username" time="8.681" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02527_Cancel_button_validation" time="24.869" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02708_MyAccount_Security_Validation" time="5.654" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02547_MyAccount_Password_Validation" time="27.892" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02529_MyAccount_Unable_To_Save_Twice" time="12.101" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_025395_MyAccount_Link_Navigation" time="23.508" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02535_MyAccount_Navigation_CBALender" time="21.909" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02901_MyAccount_SpecialChar_Validation" time="42.529" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_04435_MyAccount_BranchIDvalidation" time="4.948" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02520_MyAccount_EmailAddress_Suffix_Disabled" time="12.088" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02517_MyAccount_Field_Label" time="12.215" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02523_MyAccount_Phone_Mobile_validation" time="17.454" classname="hub.test.RegressionSuite.UserMyAccount">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//div[text()=&#039;Invalid mobile number.&#039;]&quot;}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.43.1&#039;, revision: &#039;5163bce&#039;, time: &#039;2014-09-10 16:27:33&#039;
System info: host: &#039;172-16-4-84&#039;, ip: &#039;172.16.4.84&#039;, os.name: &#039;windows&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.0&#039;, java.version: &#039;1.8.0_11&#039;
Driver info: driver.version: unknown
Command duration or timeout: 1.54 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.38.0&#039;, revision: &#039;bd32d4e7247a32d801b3d2d4f84a4e7384300378&#039;, time: &#039;2013-12-05 09:01:37&#039;
System info: host: &#039;Skylark.local&#039;, ip: &#039;10.0.10.179&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.9.5&#039;, java.version: &#039;1.7.0_04&#039;
Session ID: f730d020c9e1b58597bac8daa24537ef323b0580
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, acceptSslCerts=false, browserName=firefox, rotatable=false, locationContextEnabled=true, webdriver.remote.sessionid=f730d020c9e1b58597bac8daa24537ef323b0580, version=28.0, cssSelectorsEnabled=true, databaseEnabled=true, handlesAlerts=true, browserConnectionEnabled=true, nativeEvents=false, webStorageEnabled=true, applicationCacheEnabled=true, takesScreenshot=true}]">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[text()='Invalid mobile number.']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:33'
System info: host: '172-16-4-84', ip: '172.16.4.84', os.name: 'windows', os.arch: 'x86', os.version: '6.0', java.version: '1.8.0_11'
Driver info: driver.version: unknown
Command duration or timeout: 1.54 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.38.0', revision: 'bd32d4e7247a32d801b3d2d4f84a4e7384300378', time: '2013-12-05 09:01:37'
System info: host: 'Skylark.local', ip: '10.0.10.179', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.9.5', java.version: '1.7.0_04'
Session ID: f730d020c9e1b58597bac8daa24537ef323b0580
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, acceptSslCerts=false, browserName=firefox, rotatable=false, locationContextEnabled=true, webdriver.remote.sessionid=f730d020c9e1b58597bac8daa24537ef323b0580, version=28.0, cssSelectorsEnabled=true, databaseEnabled=true, handlesAlerts=true, browserConnectionEnabled=true, nativeEvents=false, webStorageEnabled=true, applicationCacheEnabled=true, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:307)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:404)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:299)
	at hub.library.FunctionReference.getText(FunctionReference.java:438)
	at hub.utilities.RegressionSuiteUtil.UserRegressionSuiteUtil.MyAccount_Phone_Mobile_validation(UserRegressionSuiteUtil.java:1107)
	at hub.test.RegressionSuite.UserMyAccount.RT_02523_MyAccount_Phone_Mobile_validation(UserMyAccount.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.38.0', revision: 'bd32d4e7247a32d801b3d2d4f84a4e7384300378', time: '2013-12-05 09:01:37'
System info: host: 'Skylark.local', ip: '10.0.10.179', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.9.5', java.version: '1.7.0_04'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:125)
	... 32 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[text()='Invalid mobile number.']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:33'
System info: host: '172-16-4-84', ip: '172.16.4.84', os.name: 'windows', os.arch: 'x86', os.version: '6.0', java.version: '1.8.0_11'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.38.0', revision: 'bd32d4e7247a32d801b3d2d4f84a4e7384300378', time: '2013-12-05 09:01:37'
System info: host: 'Skylark.local', ip: '10.0.10.179', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.9.5', java.version: '1.7.0_04'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Windows/proxy/anonymous7030220710661509653webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9618)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///C:/Windows/proxy/anonymous7030220710661509653webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9627)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Windows/proxy/anonymous7030220710661509653webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:11612)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Windows/proxy/anonymous7030220710661509653webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:11617)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Windows/proxy/anonymous7030220710661509653webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:11559)
]]>
    </failure>
  </testcase> <!-- RT_02523_MyAccount_Phone_Mobile_validation -->
  <testcase name="RT_02528_Blank_FirstLast_position" time="23.830" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02525_MyAccount_Save_Successfully" time="65.547" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02902_MyAccount_Logout_Link" time="11.778" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02522_MyAccount_EmailAddress_Suffix_Enabled_Invalid_message" time="12.321" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02546_Previous_Username_Can_Be_Use" time="24.714" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02518_MyAccount_EmailAddress_Suffix_Enabled" time="23.646" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02514_MyAccount_Required_Fields_Asterisk" time="62.818" classname="hub.test.RegressionSuite.UserMyAccount"/>
  <testcase name="RT_02521_MyAccount_EmailAddress_FormatValidation" time="40.279" classname="hub.test.RegressionSuite.UserMyAccount"/>
</testsuite> <!-- hub.test.RegressionSuite.UserMyAccount -->
